### PR TITLE
Cell Press: Separate out from Elsevier again

### PR DESCRIPTION
**Pull request by adam3smith**

### CODE REVIEW

**Comment by rmzelle** [comment]
Please go ahead with this if you're done; looks all good to me. Could you just regenerate the styles? I updated the datestamp in the template.

**Comment by adam3smith** [comment]
yes, will finish this tonight, thanks!

**Comment by rmzelle** [comment]
(I'm on this)

**Comment by rmzelle** [comment]
Merged the styles repo PR.

File: elsevier/_skip.txt [hunk 1]
```diff
@@ -16,9 +15,29 @@ aasri-procedia
...
+developmental-cell
+immunity
+molecular-cell
+neuron
+stem-cell-reports
+structure
+#Cell but different style
+the-american-journal-of-human-genetics
+#Cell, different style, not yet done
+molecular-plant
```
Comment (rmzelle) [inline elsevier/_skip.txt]: @damnation333, if you're up for it, we still need a custom style for the Cell journal Molecular Plant (http://www.cell.com/molecular-plant/home).

@adam3smith, do you have any notes on what is different in this style compared to that of the other Cell journals?

**Comment by damnation333** [comment]
@rmzelle 
Sure, I've noted it down

File: elsevier/_skip.txt (hunk 1 above)
Comment (adam3smith) [inline elsevier/_skip.txt]: I have this thread as my last note on Molecular plant: https://forums.zotero.org/discussion/57671/incorrect-style-for-moleculer-plant where I say

> The style is actually identical to https://www.zotero.org/styles?q=The%20Plant%20Cell

**Pull request merged**